.github/workflows/Build_runner.yml

```diff
@@ -59,8 +59,8 @@ jobs:
         # Verify and wait till latest benchmark-runner version will be updated in Pypi (timeout 900 seconds)
         timeout=900
         sleep_time=30
+        pip --no-cache-dir install setuptools benchmark-runner --upgrade
```

Comment:
Will this change what version shows up because you've moved this before expected_version rather than after, and is that an issue?

Reply:
No, it fecthes the version from checkout repository setup file and from benckmark runner installed pkg version.

Reply:
OK